common-utils/tests/deployment/unit/test_deployment.py

```diff
@@ -150,28 +148,19 @@ def test_flow_deployment(mock_cmd):
     d1 = FlowDeployment(deployment_name="test", schedule=CronSchedule(cron="0 0 * * *"))  # type: ignore
     x1 = d1._get_schedule_arg()
     assert x1 == "--cron '0 0 * * *'"
-    d2 = FlowDeployment(deployment_name="test", schedule=IntervalSchedule(interval=timedelta(days=2)))  # type: ignore
+    d2 = FlowDeployment(deployment_name="test")  # type: ignore
     x2 = d2._get_schedule_arg()
-    assert x2 == "--interval 172800"
-    d3 = FlowDeployment(deployment_name="test", schedule=RRuleSchedule(rrule="FREQ=HOURLY;BYDAY=MO,TU,WE,TH,FR;BYHOUR=9,10,11,12,13,14,15,16,17"))  # type: ignore
-    x3 = d3._get_schedule_arg()
-    assert (
-        x3
-        == "--rrule 'FREQ=HOURLY;BYDAY=MO,TU,WE,TH,FR;BYHOUR=9,10,11,12,13,14,15,16,17'"
-    )
-    d4 = FlowDeployment(deployment_name="test")  # type: ignore
-    x4 = d4._get_schedule_arg()
-    assert x4 == ""
+    assert x2 == ""
 
     # Validate push deployment method using mock on run_command.
-    d5 = FlowDeployment(
+    d3 = FlowDeployment(
         deployment_name="test",
-        schedule=IntervalSchedule(interval=120),  # type: ignore
+        schedule=CronSchedule(cron="0 6 * * *", timezone="America/Los_Angeles"),  # type: ignore
```

Comment:
The cron schedule def seems simpler to me. Question: do we always run on Pacific vs UTC?

Reply:
no not always depends on the people depending on the data.  Ideally we all talk in  UTC.